### PR TITLE
✨ Display the billable icon as necessary

### LINKED ISSUE #82: Reflect the billable flag in the time entries log
## Description
We need to ensure that time entries (both groups and flat ones) display the little billable icon when the time entry is marked as billable

## Definition of done
- [ ] When a time entry is billable,  the view that indicates it in the log is visible
- [ ] When a time entry is not billable, the view that indicates it is hidden
- [ ] *STRETCH GOAL* For performance reasons, it might make sense to implement the billable view as a ViewStub, to prevent inflating it all the time. Let's not do it if it's not worth the hassle, though

### PR DESCRIPTION
**Pull request by pdrj**
### Summary
This hides/shows the billable icon as necessary.

### Technical considerations
Using a ViewStub, because performance.

### Why is it done like this and not some other way?
Because performance, but I've not measured this and think that it might be just a marginal gain :). 

### Relationships

Closes #82 

---
### Pull Request checklist

- [x] The code does not hinder the app's performance
- [x] The code does not introduce regressions
- [x] The code in this PR is tested thoroughly enough that changing the behavior to an incorrect one breaks tests

### Pointers
Create some TEs with billable flag set. I did: `.map { it.copy(duration = Duration.between(it.startTime, now), billable = Random.nextBoolean()) }` in `Repository` when stopping TEs.

Then, validate the ones that should have the billable icon displayed, indeed have it visible _and_ the ones that don't don't.

### Review template

- [ ] The code is thoroughly tested in well written and resilient manner
- [ ] The code is performant and leads to an optimal user experience
- [ ] The code is testable and does not cut corners for the sake of delivering
- [ ] The code considers older platforms well and will work in older devices just like it will in new ones

- [ ] The code considers light and dark theme
- [ ] The code considers RTL and does not mention left/right
- [ ] The code considers i18n
  - [ ] There are no hardcoded strings in the UI
  - [ ] All strings are included in every language we support OR there's an issue requesting new translations
- [ ] There are no hard-coded numbers referring to dimensions or font sizes
- [ ] The code considers UI elements that are similar and extracts their similarities into styles to avoid duplication

## Merge permissions
Me.

### CODE REVIEW

File: timer/src/main/res/layout/time_entries_log_item_billable_icon.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/billable_icon"
+    android:layout_width="18dp"
+    android:layout_height="18dp"
+    android:layout_marginEnd="4dp"
+    android:src="@drawable/ic_billable"
+    android:tint="@color/icon"
+    app:layout_constraintBottom_toBottomOf="@+id/duration"
```
Comment (daividssilverio) [inline timer/src/main/res/layout/time_entries_log_item_billable_icon.xml]: Are the constraint attributes necessary here?

Comment (daividssilverio) [reply timer/src/main/res/layout/time_entries_log_item_billable_icon.xml]: removing them from the file and keeping the constraint attrs in the view that actually uses the stub seems to be enough.

Comment (pdrj) [reply timer/src/main/res/layout/time_entries_log_item_billable_icon.xml]: I thought I tried that and it didn't work - I'll recheck and delete these if it works.

**Comment by pdrj** [comment]
> Why not do the same in the `TimeEntryGroupViewHolder`?

Because I somehow thought we don't display the icon for groups 😄

**Pull request merged**